src/pages/api/mongodb.js

```diff
@@ -7,11 +7,16 @@ const cache = new NodeCache({
   checkperiod: 600,
 });
 
+let client;
+
 export default async () => {
   const uri = "mongodb+srv://vercel-admin-user:sBhDli5BMb65Q2Q5@kespe.3isg2qr.mongodb.net/databasen?retryWrites=true&w=majority";
```

Comment:
⚠️ this shoult NOT be exposed on a public repo. @checkmatemarcus

Reply:
stfu